### PR TITLE
docs(protocol): update L2 deployment

### PR DESCRIPTION
**Pull request by davidtaikocha**

### CODE REVIEW

File: packages/protocol/deployments/mainnet-contract-logs-L2.md [hunk 1]
```diff
@@ -56,7 +56,8 @@
   - upgraded from `0xf961854D68368cFFc86d90AEe8a19E9781dACA3e` to `0x98C5De7670aA7d47C6c0551fAD27Bfe464A6751a..` @commit`` in @tx`0x0b5d6acc9c5b8ef193920246081ec5ce7268111acfc1dce1f058bea06f3953c7`
   - changed owner to `0xCa5b76Cc7A38b86Db11E5aE5B1fc9740c3bA3DE8` @tx`0xf68861171c602e3e75ca69e950957fcb908c7949c6df9a9ea3026c238ebb1e9c`
   - upgrade impl to `0x0893c8821Fa358D5f3630695Ce062204814359A1` @commit`1bd3285` @tx`0x4605c4ce594e996bdbdb532a9aefe4fab1ea36f7e2ef63eef56a7e8033810df3`
-  - upgrade impl to `0x8fb67c2c16dc8578b6d69bc668236924f4c1b0f7` @commit`3ae25fd` @tx`0xc77434f4e37959cdc0eac125303b78dd192d8727173373cc7a6158ca7d829dad`
+  - upgrade impl to `0x8fb67c2c16dc8578b6d69bc668236924f4c1b0f7` @commit`3ae25fd` @tx`0xd95435c742c01B0E982913BcA252173Ae96DF61d`
+  - upgrade impl to `0x8fb67c2c16dc8578b6d69bc668236924f4c1b0f7` @commit`04d8c87` @tx`0xd55e1edee71bd71b9bca0d3826f758a46d688b6ffdd4058fccefc98dbed70165`
```
Comment (Brechtpd) [inline packages/protocol/deployments/mainnet-contract-logs-L2.md:60]: I think a copy-past error here of `0xd95435c742c01B0E982913BcA252173Ae96DF61d` being pasted wronly in the previous tx hash and not in the update impl address.

Comment (davidtaikocha) [reply packages/protocol/deployments/mainnet-contract-logs-L2.md]: yep, fixed

File: packages/protocol/deployments/mainnet-contract-logs-L1.md [hunk 2]
```diff
@@ -84,6 +84,7 @@
   - upgrade impl to `0x951B7Ae1bB26d12dB37f01748e8fB62FEf45A8B5` @commit`1bd3285` @tx`0xf21f6bf720767db3bc9b63ef69cacb20340bdedfb6589e6a4d11fe082dfa7bd6`
   - upgrade impl to `0x3c326483EBFabCf3252205f26dF632FE83d11108` @commit`3ae25fd` @tx`0xc0ba6558642b93ee892bee0705dbcfb5130c53637e6266bfa5e3a6501167d6f2`
   - upgrade impl to `0xD28f2c26aD8bA88b0691F6BB41Ff021878052561` @commit`2b483de` @tx`0x0bbf7d1258c646f41a02a92a55825b1ebfd3659577d0f2b57b462f8895e23a04`
+  - upgrade impl to `0x01E7D369a619eF1B0E92563d8737F42C09789986` @commit`04d8c87` @tx``
```
Comment (dantaik) [inline packages/protocol/deployments/mainnet-contract-logs-L1.md:87]: missing the tx hash

Comment (davidtaikocha) [reply packages/protocol/deployments/mainnet-contract-logs-L1.md]: Added

**Review by dantaik — APPROVED** [review]

**Review by adaki2004 — APPROVED** [review]

**Pull request merged**